api/v1_queues.inc

```diff
@@ -0,0 +1,161 @@
+<?php
+include_once($relPath.'user_is.inc');
+include_once("exceptions.inc");
+
+// TODO(bpfoley): There is a lot of duplicated logic betweeen this file and
```

Comment:
Yeah, we should probably separate the data-gathering from the presentation and re-use the data bits here and in `stats/relese_queue.php`. I had to do this for the page tables in 81af9d4b5c614983fb1d09cc2e947d07dd136e03 for the API too and we could use a similar pattern here. These new data-specific functions probably belong in `pinc/release_queue.inc`.

Reply:
The initial data function version should probably just return all the data to the caller and the API can filter out the parts the user doesn't care about, but a future optimization could be to have the data function not generate the data that isn't used as some (like total number of projects matching the queue definition) are more DB-taxing than others. But that's an optimization step and I wouldn't bother in the MVP.